src/pages/BookDetailView/index.tsx

```diff
@@ -0,0 +1,189 @@
+                        </div>
+                      </Grid>
+                    </Grid>
+                  </Grid>
+                  <Grid item>
+                    <Grid container direction="row" spacing={5}>
+                      <Grid item>
+                        <CustomButton
+                          variant="outlined"
+                          size="large"
+                          sx={{ borderColor: "black", color: "primary" }}
+                        >
+                          <TypographyText variant="body1" component="div">
+                            Read now
+                          </TypographyText>
+                        </CustomButton>
+                      </Grid>
+                      <Grid item>
+                        <CustomButton
+                          size="large"
+                          onclick={handleClick}
+                          variant="contained"
+                          color="primary"
+                        >
+                          <TypographyText variant="body1" component="div">
+                            Finished Reading
+                          </TypographyText>
+                        </CustomButton>
+                      </Grid>
+                      <Grid item>
+                        <CustomButton
+                          variant="text"
+                          size="large"
+                          endIcon={<Icon url="/res/icons/right-arrow.png" />}
+                        >
+                          <TypographyText
+                            sx={{ color: theme.palette.textcolor.light }}
+                            variant="body1"
+                            component="div"
+                          >
+                            Send to Kindle
+                          </TypographyText>
+                        </CustomButton>
+                      </Grid>
+                    </Grid>
+                  </Grid>
+                </Grid>
+              </Grid>
+              <Grid item>
+                <CoverImage url="/res/cover images/1.png" />
+              </Grid>
+            </Grid>
+          </Grid>
+          <Grid item>
+            <CustomTabs
+              labels={["Synopsis", "Who is it for?", "About the author"]}
+              contents={[
+                <TypographyText
+                  color={theme.palette.textcolor.main}
+                  component="div"
+                  variant="body2"
+                >
+                  Beyond Entrepreneurship 2.0 (2020) updates Jim Collins and
+                  Bill Lazier's essential 1992 business handbook, Beyond
+                  Entrepreneurship for the entrepreneurs, visionaries, and
+                  innovators of today. This new edition combines the timeless
+                  business advice and strategy of the original text,
+                  supplemented with cutting-edge insights and case studies
+                  pertinent to today's business world.
```

Comment:
You can have a seperate file for this kind of static messages and then use them here.